### ISSUE TITLE
Can't deploy example DAGs without -f (GitHub Actions)

### ISSUE THREAD

**Issue by magdagultekin** [astronomer/astro-cli#585]
**Describe the bug**
A customer wanted to set up CI/CD on GitHub Actions with two branches. He copied the .yml file from our docs and pushed the code to `dev` branch. 
Although there were no import errors (`astro dev parse` resulted in `no errors detected in your local DAGs`), he was not able to deploy without adding `-f` flag. At the bottom you can see the error.

**What CLI Version did you experience this bug?**

`Astro CLI Version: 1.0.0`

**What Operating System is the above CLI installed on?**

macOS

**🪜 Steps To Reproduce**

1. `astro dev init`
2. pushed the code to `dev` branch in GitHub which triggered the deployment

**📸 Screenshots**

![image](https://user-images.githubusercontent.com/97064715/171882554-1bbe1b8f-c4bc-4986-957b-712f2ddd2b64.png)

**Comment by sunkickr** [comment]
more information:
- astro dev parse does not appear to work on github actions and apple M1
- error:
```
Error: something went wrong while parsing your DAGs: error building, (re)creating or starting project containers: Error response from daemon: pull access denied for test_bcc30b/airflow, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

- using sudo with commands appears to work with M1
- using sudo does not work with github actions. only option is still astro deploy -f
- astrocloud deploy still works with github actions

**Comment by sunkickr** [comment]
astro dev parse working in github actions:
<img width="897" alt="Screen Shot 2022-06-06 at 12 40 54 PM" src="https://user-images.githubusercontent.com/63181127/172206075-cd99784b-70f9-42ad-90a5-3e7a69c82a40.png">

astro deploy working in github actions:
<img width="1019" alt="Screen Shot 2022-06-06 at 12 43 08 PM" src="https://user-images.githubusercontent.com/63181127/172206222-83c7fa4b-ff91-437f-bda7-a3f640cdcc5d.png">

**Comment by paolaperaza** [comment]
Closing this out! Released in Astro CLI v1.0.1. Release notes: https://docs.astronomer.io/astro/cli/release-notes#bug-fixes
